### PR TITLE
add springcloud-eureka-feign-mybatis-seata sample

### PR DESCRIPTION
**Pull request by lightClouds917**
1.Add a demo about springcloud-eureka-feign-mybatis-seata with a intact ReadMe.md；
2.I've tested it myself；
3.It has intact ReadMe.md,hope to help beginners;

### CODE REVIEW

File: springcloud-eureka-feign-mybatis-seata/order-server/pom.xml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<version>1.18.8</version>
+		</dependency>
+
+		<!--seata-->
+		<dependency>
+			<groupId>com.alibaba.cloud</groupId>
+			<artifactId>spring-cloud-alibaba-seata</artifactId>
+			<version>2.1.0.RELEASE</version>
```
Comment (zjinlei) [inline springcloud-eureka-feign-mybatis-seata/order-server/pom.xml]: Exclude dependency seata version 071

File: springcloud-eureka-feign-mybatis-seata/storage-server/pom.xml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<version>1.18.8</version>
+		</dependency>
+		<!--seata-->
+		<dependency>
+			<groupId>com.alibaba.cloud</groupId>
+			<artifactId>spring-cloud-alibaba-seata</artifactId>
+			<version>2.1.0.RELEASE</version>
```
Comment (zjinlei) [inline springcloud-eureka-feign-mybatis-seata/storage-server/pom.xml]: Exclude dependency seata version 071

File: springcloud-eureka-feign-mybatis-seata/account-server/pom.xml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<version>1.18.8</version>
+		</dependency>
+		<!--seata-->
+		<dependency>
+			<groupId>com.alibaba.cloud</groupId>
+			<artifactId>spring-cloud-alibaba-seata</artifactId>
+			<version>2.1.0.RELEASE</version>
```
Comment (zjinlei) [inline springcloud-eureka-feign-mybatis-seata/account-server/pom.xml]: Exclude dependency seata version 071

File: springcloud-eureka-feign-mybatis-seata/order-server/pom.xml [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+		</dependency>
+		<dependency>
+			<groupId>io.seata</groupId>
+			<artifactId>seata-all</artifactId>
+			<version>0.8.0</version>
```
Comment (zjinlei) [inline springcloud-eureka-feign-mybatis-seata/order-server/pom.xml]: Unified version number to parent project management

Comment (lightClouds917) [reply springcloud-eureka-feign-mybatis-seata/order-server/pom.xml]: Ok, I'll deal with them all

Comment (lightClouds917) [reply springcloud-eureka-feign-mybatis-seata/order-server/pom.xml]: done

**Pull request merged**